### PR TITLE
feat(SCM): add a datasource to manage SCM certificate

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:
add a datasource to manage SCM certificate

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/scm' TESTARGS='-run=TestAccDatasourceCertificate_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/scm -v -run=TestAccDatasourceCertificate_basic -timeout 360m -parallel 4
=== RUN   TestAccDatasourceCertificate_basic
=== PAUSE TestAccDatasourceCertificate_basic
=== CONT  TestAccDatasourceCertificate_basic
--- PASS: TestAccDatasourceCertificate_basic (16.64s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/scm       16.702s
```

### CODE REVIEW

File: docs/data-sources/scm_certificates.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `status` - (Optional, String) Certificate status.  
+The options are as follows:
+  - ALL: All certificate statuses.
```
Comment (niuzhenguo) [inline docs/data-sources/scm_certificates.md]: `status` seems more common for the plural of status.

File: docs/data-sources/scm_certificates.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+The options are as follows:
+  - SINGLE_DOMAIN: Single domain names
+  - WILDCARD: Wildcard domain names
+  - MULTI_DOMAIN: Multiple domain names
+
+* `validity_period` - Certificate validity period, in months.
+
+* `status` - Certificate status.  
+The options are as follows:
+  - ALL: All certificate statuses.
```
Comment (niuzhenguo) [inline docs/data-sources/scm_certificates.md]: ditto.

Comment (hmbig2) [reply docs/data-sources/scm_certificates.md]: done

File: docs/data-sources/scm_certificates.md (hunk 1 above)
Comment (hmbig2) [reply docs/data-sources/scm_certificates.md]: OK

**Pull request merged**